.github/workflows/build.yaml

```diff
@@ -90,7 +90,7 @@ jobs:
   wheel-build-dask-cudf:
     needs: wheel-publish-cudf
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-24.04
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@test-cuda-12.2
     with:
       matrix_filter: map(select(.ARCH == "amd64" and .PY_VER == "3.10" and (.CUDA_VER == "11.8.0" or .CUDA_VER == "12.0.1")))
```

Comment:
Should `CUDA_VER` read `12.2.2` instead?

Reply:
It doesn't really matter, either will work. We just need _one_ CUDA 11 and _one_ CUDA 12 version to do the publishing job. But I'll update it to align with the latest version (12.2.2 instead of 12.0.1).

Reply:
```suggestion
      matrix_filter: map(select(.ARCH == "amd64" and .PY_VER == "3.10" and (.CUDA_VER == "11.8.0" or .CUDA_VER == "12.2.2")))
```